frontend/lib/src/components/shared/BaseButton/styled-components.ts

```diff
@@ -224,16 +227,20 @@ export const StyledSecondaryFormSubmitButton = styled(
 export const StyledIconButton = styled(
   StyledBaseButton
 )<RequiredBaseButtonProps>(({ size, theme }) => {
-  const iconPadding: Record<BaseButtonSize, string> = {
-    [BaseButtonSize.XSMALL]: theme.spacing.threeXS,
-    [BaseButtonSize.SMALL]: theme.spacing.twoXS,
-    [BaseButtonSize.MEDIUM]: theme.spacing.md,
-    [BaseButtonSize.LARGE]: theme.spacing.lg,
+  let minWidth = "12rem"
+  if (size === BaseButtonSize.SMALL) {
+    minWidth = "8rem"
+  } else if (size === BaseButtonSize.LARGE) {
+    minWidth = "20rem"
   }
```

Comment:
I have removed it for now completely because this was still a left-over artifact from the prototype phase where we experimented with whether we should allow different sizes. Thanks for bringing it to my attention 🙂